core/src/gas_station.rs

```diff
@@ -6,8 +6,19 @@ use isahc::http::uri::Uri;
 use serde::Deserialize;
 use uint::FromDecStrErr;
 
-/// The default uri at which the gas station api is available under.
-pub const DEFAULT_URI: &str = "https://safe-relay.gnosis.io/api/v1/gas-station/";
+/// The default uris at which the gas station api is available under.
+const DEFAULT_MAINNET_URI: &str = "https://safe-relay.gnosis.io/api/v1/gas-station/";
+const DEFAULT_RINKEBY_URI: &str = "https://safe-relay.rinkeby.gnosis.io/api/v1/gas-station/";
+
+pub fn api_url_from_network_id(network_id: u64) -> &'static str {
```

Comment:
ubernit: I would slightly prefer returning an `Option<&'static str>` and then `.unwrap()`-ing at the call-site.